.github/workflows/full-ci.yml

```diff
@@ -311,7 +311,7 @@ jobs:
             os: ubuntu-20.04
             artifact-name: linux-nightly
             godot-binary: godot.linuxbsd.editor.dev.x86_64
-            rust-extra-args: --features godot/api-custom,godot/experimental-threads,godot/serde,codegen-full-experimental
+            rust-extra-args: --features godot/api-custom,experimental-threads,godot/serde,codegen-full-experimental
```

Comment:
Could you order it so that the `itest` features come first, and `godot/` ones later?

Maybe it's more explicit to prefix the former with `itest/` as well?

(same below)